modules/api.py

```diff
@@ -35,8 +38,8 @@
     Tag,
     TimelineEventType,
     Type,
+    ProxyType as SettingsProxyType,
```

Comment:
the name clashing can be avoided by using top-level module imports, like `python_socks.ProxyType`